Autopilot/Test/Inc/airspeed_Mock.hpp

```diff
@@ -0,0 +1,23 @@
+/*
+*
+* A header for an airspeed sensor mock
+*
+*/
+
+#ifndef AIRSPEED_MOCK_H
+#define AIRSPEED_MOCK_H
+
+#include "airspeed.hpp"
+
+#include <gmock/gmock.h>
+
+
+class MockAirspeed : public airspeed {
+	public:
+
+		MOCK_METHOD(void, Init, ());
+		MOCK_METHOD(void, BeginMeasuring, ());
+		MOCK_METHOD(void, GetResult, (airspeedData_t *Data));
+};
+
+#endif
```

Comment:
There should be a new line at the end of this file. (C++ standards prior to C++ 11 recognize no new line as a problem, as do posix standards).